tests/testsuite/features.rs

```diff
@@ -280,6 +280,7 @@ fn dependency_activates_typoed_feature() {
 versions that meet the requirements `*` are: 0.0.1
 
 package `foo` depends on `bar` with feature `bar` but `bar` does not have that feature.
+ package `bar` does have feature `baz`
```

Comment:
Is the leading space intentional?

Reply:
Other clauses have a context line like this with a single leading space.

Its weird.  It seems like it should be one of 0, 2, or 4 spaces but I didn't feel like following up what the intent was.

Reply:
Merging now to get the incremental improvement rather than blocking on figuring out what the intended indentation is.

Reply:
@Eh2406 when back from vacation, if you'd be willing to look at these errors and let us know what you think would work well, I'd appreciate it.